main/llvm/APKBUILD

```diff
@@ -0,0 +1,252 @@
+# Contributor: Martell Malone <martell@marinelayer.io>
+# Contributor: Travis Tilley <ttilley@gmail.com>
+# Contributor: Mitch Tishmack <mitch.tishmack@gmail.com>
+# Contributor: Jakub Jirutka <jakub@jirutka.cz>
+# Maintainer: Jakub Jirutka <jakub@jirutka.cz>
```

Comment:
@martell have you asked if @jirutka is ok with being maintainer for llvm7?

Reply:
He did not. I don’t wanna maintain LLVM packages.

Reply:
see https://github.com/alpinelinux/aports/pull/5717#issuecomment-474686094